modules/EmailAPIs.py

```
@@ -195,10 +206,32 @@ def open_mail(self, id):
 
+class FakeMailAPI:
+    def __init__(self, driver: Chrome):
+        self.class_name = 'fakemail'
+        self.driver = driver
+        self.email = None
+        self.window_handle = None
+
+    def init(self):     
+        self.driver.get('https://www.fakemail.net')
+        self.window_handle = self.driver.current_window_handle
+        untilConditionExecute(self.driver, f'return {GET_EBID}("email").innerText != null')
```

Ruff

`untilConditionExecute` may be undefined, or defined from star imports

(F405)